book.rb

```diff
@@ -11,4 +11,16 @@ def initialize(title, author)
   def to_s
     "Title: \"#{@title}\", Author: #{author}"
   end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'title' => @title,
+      'author' => @author
+    }.to_json(*args)
+  end
+
+  def self.json_create(object)
+    new(object['title'], object['author'])
+  end
```

Comment:
The data is not persisted when I exit the app, it shows empty array when I check `books.json`

- [ ] Please fix this issue for every other file type

Reply:
@AshakaE  did you create a new book and it didn't persist?